client_plugin/Makefile

```diff
@@ -319,6 +322,8 @@ deploy-vm-test:
 	$(DEPLOY_VM_TEST_SH) "$(VMS)" $(BIN) $(SCRIPTS)
 deploy-vm-plugin:
 	$(DEPLOY_VM_PLUGIN_SH) "$(VM2)" $(PLUGIN_BIN) $(MANAGED_PLUGIN_LOC) $(SCRIPTS) $(DOCKER_HUB_REPO) $(VERSION_TAG) $(EXTRA_TAG)
+deploy-plugin-windows-vm:
```

Comment:
@shuklanirdesh82 came up with `deploy-plugin-windows-vm` and we agreed on that because it was sounding more intuitive. Should it still be changed?